### ISSUE TITLE
[FEATURE] Actually add MatchTournamentContext info in MatchSeries/Tournament GraphQL data

### ISSUE THREAD

**Issue by Wiethoofd** [Challengermode/public-api-issues#10]
Please add more/specific definitions to [TournamentRound](https://www.challengermode.com/developers/docs/public-api/reference/definitions#tournamentround) when implementing it to the GraphQL data and add the actual round as either name or ENUM/int mappable definitions, similar to [BracketType](https://www.challengermode.com/developers/docs/public-api/reference/definitions#brackettype). (these are links to legacy REST API docs, which 404 on direct request)

There is currently no way to figure out from a match in the GraphQL if it is a Round 1 game, Semifinals or bronze match in the final round of a Tournament.

From a tournament I am able to get this response for the stages with format `101` (*RoundRobinSingleElimination*), showing 'stageIndex 1' here, which in turn gives me 2 rounds (which I know are semifinal and finals, in that order) with 2 matchSeries per round.
![image](https://github.com/Challengermode/challengermode-api-resources/assets/9917564/279207cd-e904-4c03-8bb5-0901a9f1e453)
The supposed '[MatchTournamentContext](https://www.challengermode.com/developers/docs/challengermode-api/reference/definitions#matchtournamentcontext)' the graphql sdl lists which should provide a 'stageNumber' and a 'roundNumber' seems not implemented, or requires custom queries to be provided as data in the output.

The stage/round numbers would be integers, which have no definitions in the documentation or SDL currently.

Note that the info provided for TournamentFormat for the [GraphQL](https://www.challengermode.com/developers/docs/challengermode-api/reference/definitions#tournamentformat) is completely different and unusable compared to the legacy defintion for the [REST API](https://www.challengermode.com/developers/docs/public-api/reference/definitions#tournamentformat); the 101 I referenced earlier and is shown in the output of a graphql response is only in the legacy public rest api docs, and not provided as `SINGLE_ELIMINATION` or even `ROUND_ROBIN` anywhere in the data!

TLDR: graphql sdl and doesn't match actual data in served response and documentation is vert incomplete, especially in comparison to the (now legacy) REST API docs.
![image](https://github.com/Challengermode/challengermode-api-resources/assets/9917564/8fec4f3c-40c4-4f24-9912-730ac34c9869)
